src/sections/FAQs.tsx

```diff
@@ -60,7 +60,7 @@ const Faq = () => {
               onClick={() => toggleFaq(index)}
             >
               <div className='flex gap-3 items-center'>
-                {/* blue diagonal icon  */}
+                {/* blue diagonal diamond icon  */}
                 <div className="w-3 h-3 rotate-[-45deg] rounded-[2.268px] border border-[#2962A5] bg-[#062E5D]"></div>
```

Comment:
Since this is a repeating component, you can create a separate component from it and use it everywhere. This approach makes it easier to understand which component is being used, and you won't have to copy this complex line (with lots of colors and other elements) everywhere